doit/dependency.py

```diff
@@ -409,21 +475,16 @@ def save_success(self, task):
                 self._set(task.name, "result:", get_md5(task.result))
 
         # file-dep
+        self._set(task.name, 'checker:', self.checker.__class__.__name__)
```

Comment:
Do you think it is possible (and desirable) to not save any name for the default `md5`?
I am afraid of some bloat in the `.doit.db`